homeassistant/components/remote/lg_netcast.py

```diff
@@ -0,0 +1,108 @@
+"""
+Remote control support for the LG Netcast TV.
+
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/remove.lg_netcast/
+"""
+import time
+from requests import RequestException
+import voluptuous as vol
+from homeassistant.components.remote import (RemoteDevice, PLATFORM_SCHEMA,
+                                             ATTR_NUM_REPEATS, ATTR_DELAY_SECS,
+                                             DEFAULT_DELAY_SECS)
+from homeassistant.const import (CONF_HOST, CONF_NAME,
+                                 CONF_ACCESS_TOKEN, DEVICE_DEFAULT_NAME,
+                                 STATE_OFF)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['pylgnetcast-homeassistant==0.2.0.dev0']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_NAME, default=DEVICE_DEFAULT_NAME): cv.string,
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_ACCESS_TOKEN):
+        vol.All(cv.string, vol.Length(max=6)),
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the LG Netcast remotes."""
+
+    from pylgnetcast import LgNetCastClient
+    host = config.get(CONF_HOST)
```

Comment:
Use `dict[key]` for required config keys and keys with default config schema values.